pkg/frontend/frontend.go

```diff
@@ -14,15 +14,18 @@ import (
 // Server contains frontend related OPI services
 type Server struct {
 	pb.FrontendNvmeServiceServer
+	pb.FrontendVirtioBlkServiceServer
 
 	nvme *frontend.NvmeParameters
 	rpc  spdk.JSONRPC
 }
 
 // NewServer creates initialized instance of Nvme server
 func NewServer(jsonRPC spdk.JSONRPC) *Server {
-	opiSpdkServer := frontend.NewServerWithSubsystemListener(jsonRPC, NewSubsystemListener())
+	opiSpdkServer := frontend.NewCustomizedServer(
+		jsonRPC, NewSubsystemListener(), NewMevBlkTransport())
```

Comment:
I like `NewMevBlkTransport` , can we do the same for `NewSubsystemListener` ? I mean prefix with `Mev` or `Npi` ?

Reply:
Created issue https://github.com/opiproject/opi-intel-bridge/issues/241
I am thinking to rename `SubsystemListener` to `NvmeTransport` in SPDK bridge to be consistent. Also, I like the transport in name... WDYT?